src/build_test.cc

```diff
@@ -1750,15 +1800,17 @@ TEST_F(BuildTest, RspFileSuccess)
   EXPECT_TRUE(builder_.Build(&err));
   ASSERT_EQ(3u, command_runner_.commands_ran_.size());
 
-  // The RSP files were created
-  ASSERT_EQ(files_created + 2, fs_.files_created_.size());
+  // The RSP files and temp file to acquire output mtimes were created
+  ASSERT_EQ(files_created + 3, fs_.files_created_.size());
   ASSERT_EQ(1u, fs_.files_created_.count("out 2.rsp"));
   ASSERT_EQ(1u, fs_.files_created_.count("out 3.rsp"));
+  ASSERT_EQ(1u, fs_.files_created_.count(".ninja_tmp"));
 
-  // The RSP files were removed
-  ASSERT_EQ(files_removed + 2, fs_.files_removed_.size());
+  // The RSP files and temp file to acquire output mtimes were removed
+  ASSERT_EQ(files_removed + 3, fs_.files_removed_.size());
   ASSERT_EQ(1u, fs_.files_removed_.count("out 2.rsp"));
   ASSERT_EQ(1u, fs_.files_removed_.count("out 3.rsp"));
+  ASSERT_EQ(1u, fs_.files_removed_.count(".ninja_tmp"));
```

Comment:
This should definitely be part of the test though. Maybe fs_ should have a method to count files and we should check that everything was cleaned up.